### ISSUE TITLE
How to onboard new seller using route api

### ISSUE THREAD

**Issue by avinashk369** [razorpay/razorpay-flutter#244]
I am working on a marketplace application and it has a common feature like seller list their products on the application, user make a purchase of product and application will split the payment to the seller and take its platform fees. i have checked all the razorpay api, be it route/razorpayx/partner. i assume that route api will be the good fit to cover the use case but route api has no api to create linked accounts, only option they has is from their dashboard and i believe this wouldn't be the ideal case of seller onboarding. i am kind of stuck into the docs from couple of days. i couldn't find any sample app or even a question on stackoverflow about the same.
Any insights or suggestion would be highly appreciated

**Comment by hetthummar** [comment]
Hey avinashk369,
I also faced this problem but unfortunately, there is no API for creating a linked account 
[https://razorpay.com/docs/payments/route/linked-account/](url)

So what we are currently doing is when the seller submits bank account detail we show them the message that please wait for 24 hours until we review your details, then we check the seller's detail through our dashboard and upload that detail manually to Razorpay and then we add account_id given by Razorpay to our database.

This is not a solution but this is the only way until Razorpay made a separate API for adding sellers.

**Comment by avinashk369** [comment]
> Hey avinashk369, I also faced this problem but unfortunately, there is no API for creating a linked account [https://razorpay.com/docs/payments/route/linked-account/](url)
> 
> So what we are currently doing is when the seller submits bank account detail we show them the message that please wait for 24 hours until we review your details, then we check the seller's detail through our dashboard and upload that detail manually to Razorpay and then we add account_id given by Razorpay to our database.
> 
> This is not a solution but this is the only way until Razorpay made a separate API for adding sellers.

Hey,
Thanks for the info. Actually they have separate api to create linked account while merchant onboarding.
I share my concern with razorpay tech support team and they gave me the api details 
https://razorpay.com/docs/v2/beta/api/payments/route/account-apis-beta/
hope this helps you.

**Comment by hetthummar** [comment]
Thanks, @avinashk369 
it will help me a lot.

**Comment by rajeshjoshi8792** [comment]
Hii,

i m working on the same. i have 1 question regarding creating a linked account.

i have enabled test mode. and i m not able to find any test account details to use in test mode. so is there any test account details available in test mode or we have to use real account details?

**Comment by shubham49** [comment]
@avinashk369 @hetthummar do you know any way to update bank account details via API?

**Comment by ronitsalvi** [comment]
@shubham49  Did you find a way to add/ update bank account details via API?

**Comment by ronitsalvi** [comment]
I think Accounts APIs can be used- https://razorpay.com/docs/api/payments/route/account-apis-beta/#create-an-account

This API has the object bank_account through which a user can pass ifsc code, account no. etc. @shubham49

**Comment by akshsharma12** [comment]
How you collect the user details by own Ui page

**Comment by myatinsharma** [comment]
It is so frustrating for me, to find solutions for such scenarios from Razorpay. They are not even contacting. It's confusing, I will create a linked account then where I can add bank account details in the api `https://razorpay.com/docs/api/payments/linked-account-onboarding/create-linked-account/`? In body, they are not asking for it and in Error they are giving errors for it. (Do not worry, I checked 10 times and I did not see any param regarding bank detail in body).

**Comment by myatinsharma** [comment]
It is so frustrating for me, to find solutions for such scenarios from Razorpay. They are not even contacting. It's confusing, I will create a linked account then where I can add bank account details in the api `https://razorpay.com/docs/api/payments/linked-account-onboarding/create-linked-account/`? In body, they are not asking for it and in Error they are giving errors for it. (Do not think that I did not check well. I checked more than 10 times and I did not see any param regarding bank detail in body).
